samples/csharp_dotnetcore/57.teams-conversation-bot/Bots/TeamsConversationBot.cs

```diff
@@ -55,11 +56,15 @@ protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivi
                 await CardActivityAsync(turnContext, false, cancellationToken);
         }
 
-        protected override async Task OnTeamsMembersAddedAsync(IList<TeamsChannelAccount> membersAdded, TeamInfo teamInfo, ITurnContext<IConversationUpdateActivity> turnContext, CancellationToken cancellationToken)
```

Comment:
InstallationUpdate is different from members added. Should OnInstallationUpdateActivityAsync be additive, rather than replacing OnTeamsMembersAddedAsync?